public/app/features/dashboard/panel_editor/HelpToggle.tsx

```diff
@@ -0,0 +1,21 @@
+import { Icon } from '@grafana/ui';
+import { css, cx } from 'emotion';
+import React, { useState } from 'react';
+
+const helpStyle = css`
+  margin-top: 4px;
+`;
+
+export const HelpToggle: React.FunctionComponent = ({ children }) => {
```

Comment:
nit: feels like this pattern exists on more places? Is this something that we could break out into a grafana/ui component?

Reply:
Hm, I cannot recall any other places we use this pattern tbh. We use something similar which is the InfoBox, but it serves slighlty different purpose I think.